docs/other/monitor-kubernetes-kuantifier.md

```diff
@@ -0,0 +1,191 @@
+    
+    - A fully configured `.processor.config` might look like:
+
+           :::yaml
+           processor:
+             config:
+               NAMESPACE: workload-namespace
+               SITE_NAME: CHTC
+               VO_NAME: University of Wisconsin
+               SUBMIT_HOST: tiger-cluster.chtc.wisc.edu
+               PROMETHEUS_SERVER: prometheus-server.monitoring.svc.cluster.local
+
+1. (Optional) If Prometheus in your cluster is configured to require authentication, an
+   authentication header can be specified via a key within an already-existing [Secret](kubernetes-secret) in the namespace:
+
+       :::yaml
+       processor:
+         prometheus_auth:
+           secret: <secret name>
+           key: <key in secret containing auth header>
+
+1. (Optional) Update the frequency of the Kuantifier Reporting job. This may be useful for debugging.
+
+       :::yaml
+       cronJob:
+         schedule: "@daily"
+
+### Installing Kuantifier
+
+After configuring an appropriate values file for your instance, install the chart via helm:
+
+    :::console
+    helm install -f <values.yaml> -n <install namespace> kuantifier oci://hub.opensciencegrid.org/iris-hep/kuantifier
+
+Validation
+----------
+
+After running helm install, ensure that the expected kubernetes objects have been created. The following commands assume
+that kuantifier has been installed in the monitoring namespace.
+
+1. Check that a CronJob was created for running the kuantifier processor:
+
+       :::console
+       kubectl -n monitoring get cronjob kuantifier-cronjob
+
+1. Check that a ConfigMap was created to configure processor jobs, and that the values in the ConfigMap
+   align with the values set in `.processor.config` in the values file:
+
+       :::console
+       kubectl -n monitoring get configmap kuantifier-processor-config -o yaml
+
+
+If the helm chart artifacts are present as expected, run a test instance of the CronJob and inspect its output.
+
+1. Create a new job from the CronJob, then find the Pod created by the job
+
+       :::console
+       kubectl -n monitoring create job --from=cronjob/kuantifier-cronjob kuantifier-test-job
+       kubectl -n monitoring get pod | grep kuantifier-test-job
+
+1. Inspect the logs from the processor initContainer, which queries prometheus to generate output records.
+
+       :::console
+       kubectl -n monitoring logs <test-job-pod-name> -c processor
+
+1. Inspect the logs from the gratia-output container, which sends the output records to GRACC.
+
+       :::console
+       kubectl -n monitoring logs <test-job-pod-name> -c gratia-output
+
+If both the processor initContainer and gratia-output container run to completion without error, the next step
+is to confirm with a member of the OSG technology team that the results are visible in GRACC.
```

Comment:
I think we should just point them at the https://gracc.opensciencegrid.org/d/000000079/site-summary?orgId=1&var-site=Purdue%20Geddes&var-type=Batch&var-interval=$__auto_interval_interval dashboard and have them self service this. It may take a few hours for results to show up (Ashton/Derek would know for sure)

That being said, there are lots of places where this documentation can go wrong. Please add our standard `Getting Help` section